pkg/netceptor/conn.go

```diff
@@ -300,17 +300,40 @@ func (li *Listener) Addr() net.Addr {
 	return li.pc.LocalAddr()
 }
 
+type QuicStreamForConn interface {
```

Comment:
How do you feel about putting interface definitions at the top of the file? 

Would you think it would be more or less readable? 
Personally I find it more readable but I would like more opinions.

Reply:
I agree it's more readable to have types at the top, moved in 65e94c06396dc1c20af35bb6029e75642fe24d52.